### ISSUE TITLE
How can I evaluate the model on BLEU score?

### ISSUE THREAD

**Issue by BarakBat** [jadore801120/attention-is-all-you-need-pytorch#85]
Hi, How can I evaluate the model on BLEU score? and when training printing accuracy, what is the benchmark that this determines by?

Thanks.

**Comment by vgaraujouc** [comment]
You should execute [translate.py](https://github.com/jadore801120/attention-is-all-you-need-pytorch/blob/master/train.py), then you may use this page [**Interactive BLEU score evaluator
**](https://www.letsmt.eu/Bleu.aspx) to compare the text file generated with the ground truth.
